### PR TITLE
Added new Commissions problem in Finance labs

### PR DESCRIPTION
**Pull request by kavanparam**
- Added model for the question type (including personName, commission %, price, numberOfSales) in Question.ts
- Added FINANCE_COMMISSION_PROBLEM to the list of questions in question.ts 
- Added description for question in questionTypes.ts
- Created CommissionQuestion.tsx component as main container for the contents of the commission word problem

image

### CODE REVIEW

File: components/finance/CommissionQuestion.tsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    if (e.charCode === 13) {
+      onSubmit();
+    }
+  };
+
+  return(
+    <div className="flex flex-col items-center gap-5">
+
+      <p>
+        {personName} earns <b>{commission}%</b> interest for every contract they sell.
```
Comment (vithushan19) [inline components/finance/CommissionQuestion.tsx:49]: should you change the word contract to say `house`?

File: components/finance/CommissionQuestion.tsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+      onSubmit();
+    }
+  };
+
+  return(
+    <div className="flex flex-col items-center gap-5">
+
+      <p>
+        {personName} earns <b>{commission}%</b> interest for every contract they sell. 
+        If they sell <b>{numberOfSales}</b> house(s) for <b>${price}</b>, how much commission
```
Comment (vithushan19) [inline components/finance/CommissionQuestion.tsx:50]: I think you should you should add the word `each` If they sell X houses for $Y each, how much commsion did they earn?

File: pages/api/labs/finance/questionGenerator.ts [hunk 3]
```diff
@@ -32,26 +32,44 @@ export const generateQuestionForFinanceSkill = (skill: Skill): Question => {
...
+      return {
+        questionType: QuestionType.FINANCE_COMMISSION_PROBLEM,
+        commisionModel: {
+          personName: _personName,
+          commission: _commission,
+          price: _price,
+          numberOfSales: _numberOfSales,
+          image1: image1,
+        },
+        answer: (_numberOfSales*_price*(_commission/100)).toFixed(2).toString(),
```
Comment (vithushan19) [inline pages/api/labs/finance/questionGenerator.ts:71]: nice!

**Review by vithushan19 — APPROVED** [review]
looks good to me, just left some minor suggestions about wording. feel free to merge this in after you fix those issues.

File: components/finance/CommissionQuestion.tsx (hunk 2 above)
Comment (kavanparam) [reply components/finance/CommissionQuestion.tsx:50]: fixed!

File: components/finance/CommissionQuestion.tsx (hunk 1 above)
Comment (kavanparam) [reply components/finance/CommissionQuestion.tsx:49]: fixed!

**Pull request merged**